AddLang.feature

```diff
@@ -0,0 +1,12 @@
+﻿Feature: Create

```

Comment:
It's overkill to create a feature file for every single feature. 

You could ask yourself; what am I testing? you are testing some functionalities in the Profile page. Therefore you can create a class to test features in profile page.

Please read up on specflow and check the examples

Reply:
Wouldn't it be nice to create a feature file for each element I am testing. Example: one feature file each for language, Education, skills, certificate. I mean like how they they have already given one feature file for login so that all test cases about login functionality could be included in that Specflow feature file in future. In that case we could also run  tests for each pages separately in test explorer.